vita3k/packages/include/packages/sce_types.h

```diff
@@ -651,6 +651,7 @@ class SceRIF {
 };
 
 void register_keys(KeyStore &SCE_KEYS, int type);
+void extract_exfat(const fs::path &partition_path, const std::string &partition, const fs::path &pref_path);
```

Comment:
Why it's here? This function already defined in `packages/exfat.h`

Reply:
Oupsy 😐